### PR TITLE
[WIPTEST] containers projects summary fixes to accomodate CMP-9523

### PR DESCRIPTION
**Pull request by pavelzag**
{{pytest: cfme/tests/containers/test_containers_projects_summary.py -v --use-provider cm-env2 }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 225dec0edf2f50a0b2048e562581c7b0ecdf440a:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5515073cc8f48680f157c843003f34c166404b08:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 7914256f8c5e5941a1592bb918c99ec98dc2949a:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 7983d15b7294d33370735cf86e0dd5cae5f4e12d:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a1678f89ace8929696b4d3ef5f736e44ab3fc5e1:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d5a77c900530c6dc3184005441c16112498027f4:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b39aec6ba263dece33ac4258a8ffac8384438dda:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 78a31f4b0a081aad2ae5f7b084566f4afe391f5e:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 42cc0e12c73f1c557e594821596dcdce33efa4fc:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit afc3333419dae37ec03ee809453c31b79ba82559:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 114116de60693c029cd0a5e58a7fc679f57d2196:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/containers/test_containers_projects_summary.py [hunk 1]
```diff
@@ -45,9 +51,13 @@ def test_containers_projects_summary_relationships(provider):
...
         """
     navigate_to(Project, 'All')
-    project_name = [r.name.text for r in list_tbl.rows()]
-    for name in project_name:
+    project_names = [r.name.text for r in list_tbl.rows()]
+    for default_project in default_projects:
+        assert default_project in project_names
+    for name in default_projects:
         for field in config_option():
+            print('the tested project is: ' + name)
```
Comment (psav) [inline cfme/tests/containers/test_containers_projects_summary.py]: Please remove prints

File: cfme/tests/containers/test_containers_projects_summary.py [hunk 2]
```diff
@@ -36,7 +42,7 @@ def config_option():
 
 
 def test_containers_projects_summary_relationships(provider):
-    """ Relationships fields tests in Project summary
+    """ Relationships   fields tests in Project summary
```
Comment (psav) [inline cfme/tests/containers/test_containers_projects_summary.py]: Remove added space here

**Comment by dajoRH** [comment]
Lint report for commit dedc1e8ce51a8e319a0e4a0517d1f948126a6e7a:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by pavelzag** [comment]
@psav prints removed, please re-review.
Thanks!

File: cfme/tests/containers/test_containers_projects_summary.py [hunk 3]
```diff
@@ -45,8 +51,10 @@ def test_containers_projects_summary_relationships(provider):
             Relationships fields
         """
     navigate_to(Project, 'All')
-    project_name = [r.name.text for r in list_tbl.rows()]
-    for name in project_name:
+    project_names = [r.name.text for r in list_tbl.rows()]
```
Comment (psav) [inline cfme/tests/containers/test_containers_projects_summary.py]: If you've made ```default_projects``` a set, you can now do
```assert default_projects.is_subset(set(project_names))``` and skip out on the for loop

File: cfme/tests/containers/test_containers_projects_summary.py [hunk 4]
```diff
@@ -62,8 +70,10 @@ def test_containers_projects_summary_properties(provider):
             * Go through each Container Project in the menu and check validity of Properties fields
         """
     navigate_to(Project, 'All')
-    project_name = [r.name.text for r in list_tbl.rows()]
-    for name in project_name:
+    project_names = [r.name.text for r in list_tbl.rows()]
```
Comment (psav) [inline cfme/tests/containers/test_containers_projects_summary.py]: Here too

File: cfme/tests/containers/test_containers_projects_summary.py (hunk 3 above)
Comment (psav) [inline cfme/tests/containers/test_containers_projects_summary.py]: ```issubset```, sorry

**Comment by dajoRH** [comment]
Lint report for commit 1b379ba5a5b76ba6bba1fd1e44c15367dea04e7c:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/containers/test_containers_projects_summary.py [hunk 5]
```diff
@@ -45,8 +51,12 @@ def test_containers_projects_summary_relationships(provider):
             Relationships fields
         """
     navigate_to(Project, 'All')
-    project_name = [r.name.text for r in list_tbl.rows()]
-    for name in project_name:
+    project_names = [r.name.text for r in list_tbl.rows()]
+    project_names_set = set()
```
Comment (psav) [inline cfme/tests/containers/test_containers_projects_summary.py]: One more optimization here
```project_names = {r.name.text for r in list_tbl.rows()}```
Note the curly braces will cast it into a set and not a list

**Comment by jkrocil** [comment]
@pavelzag Why was this moved back to [RFR] without addressing the issues pointed out by @psav please?

**Comment by pavelzag** [comment]
By mistake surely. I'll check and reply soon

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by pavelzag** [comment]
This is not needed anymore. We have these covered with 
https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/containers/test_properties.py
and
https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/containers/test_relationships.py

**Pull request closed without merging**